src/core/segment_sinks/segment_buffers_store.ts

```diff
@@ -347,6 +360,43 @@ export default class SegmentSinksStore {
     }
     return true;
   }
+
+  public getSegmentSinksMetrics(): SegmentSinkMetrics {
+    return {
+      segmentSinks: {
+        audio: {
+          bufferType: "audio",
+          codec: this._initializedSegmentSinks.audio?.codec,
+          segmentInventory: (
+            this._initializedSegmentSinks.audio?.getLastKnownInventory() ?? []
+          ).map((chunk) => ({
+            ...chunk,
+            infos: getChunkContextSnapshot(chunk.infos),
+          })),
+        },
+        video: {
+          bufferType: "video",
+          codec: this._initializedSegmentSinks.video?.codec,
+          segmentInventory: (
+            this._initializedSegmentSinks.video?.getLastKnownInventory() ?? []
+          ).map((chunk) => ({
+            ...chunk,
+            infos: getChunkContextSnapshot(chunk.infos),
+          })),
+        },
+        text: {
+          bufferType: "text",
+          codec: this._initializedSegmentSinks.text?.codec,
+          segmentInventory: (
+            this._initializedSegmentSinks.text?.getLastKnownInventory() ?? []
+          ).map((chunk) => ({
+            ...chunk,
+            infos: getChunkContextSnapshot(chunk.infos),
+          })),
+        },
+      },
+    };
```

Comment:
So there's no difference between an empty buffer and a disabled buffer?
Could be good to have one

Reply:
yes, thanks. I have made a change so segmentInventory is undefined if the buffer is disabled.
An empty graph will now be correctly be displayed in the debug element if the inventory is empty.